modules/veteran_verification/spec/requests/service_history_request_spec.rb

```diff
@@ -7,12 +7,16 @@
 
   let(:scopes) { %w[profile email openid service_history.read] }
 
+  def headers(auth)
+    auth.merge('Accept' => 'application/json')
+  end
```

Comment:
I think it would be good to have a test in here that either removes the default `Accept: text/html` or that uses `Accept: */*` to verify the `set_default_format` method is working as intended.

Reply:
Changed the default format for the request specs to `Accept: */*`, which still passes everything.